src/routes/components/components.json

```diff
@@ -1,4 +1,19 @@
 [
+	{
+		"title": "sk-sitemap",
+		"repository": "https://github.com/jasongitmail/sk-sitemap",
+		"description": "Automatic SvelteKit sitemap that makes it impossible to forget to add your paths.",
+		"npm": "sk-sitemap",
+		"category": "Developer Experience",
+		"tags": [
+			"components and libraries",
+			"integrations",
+			"layout and structure",
+			"typescript",
+			"sitemap",
+			"xml"
```

Comment:
Those are new tags.

If there are no other components that can have it (and if it's the case, they need to be updated) I don't see why a tag that offer no alternatives should be added.

_(The list of tag is already long, I don't know if many people go to the struggle to search in it. Adding more tags without a real benefit don't seem like a good idea to reduce the issue, plus `sitemap` will be match be the search field anyway. But I can be wrong)_

Reply:
I can remove them. But would consider `sitemap` to be useful. The directory has similar `video` and  `images` tags.

Lmk what you think, will remove `xml` for now.

Reply:
@MacFJA ping on this

Reply:
Hi @MacFJA Just following up on this, seeing if we can get it merged please.

Reply:
Yeah, of course.
Sorry, I was off a couple of days and completely forgot about it.

I see that the CI is failing, can you run a `pnpm run format` to fix it ? I will merge it as soon as the CI is OK

Reply:
Thanks! Formatted with Prettier. Passes now